.circleci/config.yml

```diff
@@ -1648,7 +1576,7 @@ workflows:
           regression_path: /swirlds-platform/regression
           result_path: results/6N_6C/Performance
           config_type: "daily"
-          workflow-name: "GCP-Daily-Services-Comp-Restart-HTS-Performance-6N-6C"
+          workflow-name: "GCP-Daily-Services-Comp-Restart-Performance-Testnet-6N-6C"
```

Comment:
This test doesn't seem to exist in latest `develop`, just the `Random` and `Hotspot` versions.

Reply:
Yes @qnswirlds . It is not there. But I have put one version, in case we add any tests with public testnet state, that needs change in startUpAccount. I will remove it now. Thanks